packages/app/src/app/pages/common/UserMenu/index.tsx

```diff
@@ -23,22 +29,14 @@ function UserMenu() {
         />
       </ClickableContainer>
       {userMenuOpen && (
-        <HoverMenu onClose={() => signals.userMenuClosed()}>
+        <HoverMenu onClose={() => userMenuClosed()}>
           <Menu
-            openPreferences={() => {
-              signals.modalOpened({ modal: 'preferences' });
-            }}
-            openStorageManagement={() => {
-              signals.files.gotUploadedFiles();
-            }}
-            signOut={() => {
-              signals.signOutClicked();
-            }}
+            openPreferences={() => modalOpened({ modal: 'preferences' })}
+            openStorageManagement={files.gotUploadedFiles()}
+            signOut={signOutClicked()}
```

Comment:
@SaraVieira You've forgot to delete the function call, so this function will always be called when this component is loaded now

```suggestion
            signOut={signOutClicked}
```